lib/Charts/Tooltip.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-import d3 from 'd3';
+import * as d3Collection from 'd3-collection';
+import * as d3Selection from 'd3-selection';
+import * as d3Transition from 'd3-transition'; // eslint-disable-line no-unused-vars
```

Comment:
I think he's got it set up so that [`selection`](https://github.com/d3/d3-selection/blob/master/index.js) has everything under it, so my example replacement would work. 
If not, we're only using `event` and `select`, so you could also try
```import {event as d3Event, select as d3Select} from 'd3-selection'```, and then use `d3Select` instead of `d3Selection.select`.